src/main/kotlin/assimp/format/obj/ObjFileImporter.kt

```diff
@@ -456,6 +457,31 @@ class ObjFileImporter : BaseImporter() {
 }
 
 
+// TODO this is pretty much a copy past from gli.read(...) and should be added there
```

Comment:
@elect86 You are working on gli right? Can you add a `readFromMemory` there or should I just sent a PR there myself?

Reply:
I sent you an invite for the organization, so that you can automatically work on that without me having to invite for each single repo (I did the same for @Sylvyrfysh)